README.md

```diff
@@ -1,24 +1,170 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <br/>
 
-* Ruby version
+  <h3><b>Outdoor Adventure Booking</b></h3>
 
-* System dependencies
+</div>
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Outdoor Adventure Booking <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- Booking outdoor adventure is a platform where users can explore and book various thrilling outdoor activities and adventures. The website can offer a range of experiences such as hiking, rock climbing, kayaking, camping trips, wildlife tours, and more.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+- Ruby
+- Ruby on Rails
+- Postgresql
+
+## 🚀 Frontend Project <a name="live-demo"></a>
+
+- [Link to Frontend Project](https://github.com/bilalrajput09/Outdoor-Adventure-Bookings)
+
+-[DEPLOYMENT LINK](https://outdoor-adventures-booking.onrender.com/)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+```
+  git clone https://github.com/bilalrajput09/Outdoor_Adventure_Bookings_Backend.git
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/bilalrajput09/Outdoor_Adventure_Bookings_Backend.git
+
+### Install
+
+Install this project with:
+
+cd Outdoor-Adventure-Bookings-Backend
+bundle install
+
+### Usage
+
+To run the project, execute the following command:
+
+rails server
+
+### Run tests
+
+To run tests, run the following command:
+
+rspec
+
```

Comment:
- [ ] When I run the tests according to instructions, this happens:

![image](https://github.com/bilalrajput09/Outdoor_Adventure_Bookings_Backend/assets/57421823/c7167a6b-9214-47fd-84d4-244266b7e361)

This is because 9 of the tests are rswag tests, and rspec won't run them correctly.

In order to correct this issue, please be more specific:

```bash
rspec ./spec/model
rake rswag
```

Result:

![image](https://github.com/bilalrajput09/Outdoor_Adventure_Bookings_Backend/assets/57421823/635d96ae-31a5-4fd8-996a-3fbba69b6279)


That way, and no test will fail. 👍